guard.cpp

```diff
@@ -11,19 +11,26 @@ using namespace openpower::guard;
 
```

Comment:
**Commit message tips:**

- Try to follow the 50/72 rule for commit messages - that is, the subject line should not exceed 50 characters and the body should not exceed 72 characters. This is common practice in many projects which use Git.
```
guard -l command : Formatted the record ID to display 0x and the header space to align
```
- It will be great if you mentioned, "Why are you making changes, like issue details" instead of "How you changed" in the commit message.
- Please avoid adding the test machine name `root@rain104bmc` in the commit message.
- The `Change-Id:` is not required, it's used for the Gerrit review process, not for the pull request and, we are not using the Gerrit for the PHAL repository.

---------------------------------

Looks like the `ID` value is different when compared with the current code, `0x10000000` vs `00000001`, why is it like that?

Reply:
This is addressed. It was due to adding std::left to header.